sklearn/datasets/_arff_parser.py

```diff
@@ -368,14 +368,38 @@ def _pandas_arff_parser(
         header=None,
         na_values=["?"],  # missing values are represented by `?`
         comment="%",  # skip line starting by `%` since they are comments
+        quotechar='"',  # delimiter to use for quoted strings
```

Comment:
I think that it makes it more specific even if this is the default.
The below comment is therefore more meaningful.